reference/supervisor.md

```diff
@@ -1000,19 +1000,19 @@ void wb_supervisor_field_set_mf_string(WbFieldRef field, int index, const char *
 
-The `wb_supervisor_field_set_sf_*()` functions assign a value to a specified
+The `wb_supervisor_field_set_sf_*` functions assign a value to a specified
 single `field` (SF). The type of the field has to match with the name of the
 function used, otherwise the value of the field remains unchanged (and a warning
 message is displayed).
 
-The `wb_supervisor_field_set_mf_*()` functions work the same way as the
-`wb_supervisor_field_set_sf_*()` functions but with a multiple `field` (MF)
+The `wb_supervisor_field_set_mf_*` functions work the same way as the
+`wb_supervisor_field_set_sf_*` functions but with a multiple `field` (MF)
 argument. They take an additional `index` argument which refers to the index of
 the item in the multiple field. The type of the field has to match with the name
 of the function used and the index should be comprised between minus the total
 number of items and the total number of items minus one, otherwise the value of the field remains unchanged (and a warning message is displayed). Using a negative index starts the count from the last element of the field until the first one. Index -1 represents the last item and the first item is represented by index 0 or minus number of items.
 
-The set operations are received by Webots from possibly several supervisors running concurrently. In order to ensure reproducible simulation results, they are executed only once all set operations are received, just before advancing the simulation time. The order of execution of the set operations is defined by the order of the Supervisor nodes in the scene tree. As a consequence, if a supervisor sets the translation field of a node and immediately retrieves the absolute position of the same node using `wb_supervisor_node_get_position()`, it will actually get the previous position of the node. This is because the execution of the set operation is postponed to the beginning of the next simulation step. In order to retrieve the new position of the node, a `wb_robot_step()` call with a non-zero argument should be executed before calling `wb_supervisor_node_get_position()`.
+The set operations are received by Webots from possibly several supervisors running concurrently. In order to ensure reproducible simulation results, they are executed only once all set operations are received, just before advancing the simulation time. The order of execution of the set operations is defined by the order of the Supervisor nodes in the scene tree. As a consequence, if a supervisor sets the translation field of a node and immediately retrieves the absolute position of the same node using `wb_supervisor_node_get_position`, it will actually get the previous position of the node. This is because the execution of the set operation is postponed to the beginning of the next simulation step. In order to retrieve the new position of the node, a `wb_robot_step` call with a non-zero argument should be executed before calling `wb_supervisor_node_get_position`.
```

Comment:
In this sentences the modified functions have no `function` suffix (and no article).